app/views/claims/support/databases/reset.en.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :page_title, "Reset database" %>
+<% render "claims/support/primary_navigation", current: :settings %>
+
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(href: claims_support_settings_path) %>
+<% end %>
+
+<div class="govuk-width-container">
+  <span class="govuk-caption-l">Reset database</span>
+  <h1 class="govuk-heading-l">Are you sure you want to reset the <%= govuk_tag text: hosting_environment_phase(current_service), colour: hosting_environment_color, classes: "govuk-tag--middle" %> database?</h1>
+
+  <p class="govuk-body">This action will truncate the following tables:</p>
+
+  <%= govuk_list type: :bullet do %>
+    <%= tag.li tag.code("claims", class: "inline-code") %>
+    <%= tag.li tag.code("mentor_memberships WHERE mentor_memberships.type = 'Claims::MentorMembership'", class: "inline-code") %>
+  <% end %>
+
+  <p class="govuk-body">It will also perform the following data updates:</p>
+
+  <%= govuk_list type: :bullet, spaced: true do %>
+    <%= tag.li do %>
+      <div>Reset all grant conditions acceptances for all schools</div>
+      <div><code class="inline-code">schools.claims_grant_conditions_accepted_at = null"</code></div>
+      <div><code class="inline-code">schools.claims_grant_conditions_accepted_by_id = null"</code></div>
+    <% end %>
+  <% end %>
+
+  <%= govuk_button_to "Reset database", claims_support_database_path, warning: true, method: :delete, class: "govuk-button govuk-button--warning" %>
+
+  <%= govuk_link_to "Cancel", claims_support_settings_path, no_visited_state: true %>
+</div>
```

Comment:
The text in this file should be in locales

Reply:
This is a localised view - namely because it's a static behind-the-scenes page that wouldn't be (support) user-facing, ever. It's for development and pre-production environments only.

Reply:
I've avoided moving the text of this file into a locale file, as it is a localised view, signified with the file extension - `reset.en.html.erb`. If we need to add in a non-`en` locale for this page (Which we shouldn't - as this page is for DfE development/product team only - not Support, not end-users).

Side note, I think the only possibility of adding an additional language falls on the end-user's side. That possibly being Welsh - if the system expands to Wales.